mods/voidmulti.sc2mod/base.sc2data/gamedata/unitdata.xml

```diff
@@ -116,6 +276,7 @@
         <GlossaryStrongArray index="1" value="Ultralisk"/>
         <GlossaryStrongArray index="2" value="HighTemplar"/>
         <GlossaryWeakArray index="2" value="Tempest"/>
+        <Speed value="1.6406"/>
```

Comment:
Brood Lord move speed increased from 1.97 to 2.3 (from 1.406 to 1.6406 on normal speed)

Move speed is tiles per second. The patch notes scaled this to real time seconds as well